MapboxNavigation/FeedbackViewController.swift

```diff
@@ -165,12 +227,32 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
         progressBar.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         progressBar.bottomAnchor.constraint(equalTo: view.safeBottomAnchor).isActive = true
     }
+    
+    func sendFeedback(_ item: FeedbackItem) {
```

Comment:
Nit: `send(_:)` is the more Swiftian way of naming it these days, or `sendFeedbackItem(_:)` if there’s a conflict with another `send(_:)`.